Sources/PointFree/PageLayout.swift

```diff
@@ -112,6 +152,12 @@ func simplePageLayout<A>(_ contentView: View<A>) -> View<SimplePageLayoutData<A>
             rel(.alternate),
             title("Point-Free Episodes"),
             type(.application(.atom)),
+            ]),
+          link([
+            href(url(to: .blog(.feed(.atom)))),
+            rel(.alternate),
+            title("Point-Free Blog"),
+            type(.application(.atom)),
```

Comment:
it's legit to have multiple feed links in a page. feed readers just give you the choice when you paste pointfree.co in their UI. I went with "Point-Free Blog" instead of "Point-Free Pointers" to make it clear in feed readers what this is.

Reply:
oh this also creates a bunch of snapshot failures cause this is inserted into the layer, hence every page on the site.